web/themes/joinup/prototype/sass/base/_ordered_list.sass

```diff
@@ -6,7 +6,7 @@
 
 article
   ol
-    padding-left: 0
+    padding-left: 15px
```

Comment:
The bullet should be normally aligned and not centered.
Example:
![download](https://user-images.githubusercontent.com/1923476/28827481-fe2c3be4-76cd-11e7-940e-84ec52d0bf8c.png)